### PR TITLE
Notifications

### PR DESCRIPTION
**Pull request by jakekinsella**
Trello Card: https://trello.com/c/ljVqyNnJ/72-get-notification-libraries-together-and-create-one-method-that-will-send-a-notification-using-the-correct-library-for-your-os

### CODE REVIEW

File: client/src/main/java/org/robockets/runqueue/client/notifications/MacNotification.java [hunk 1]
```diff
@@ -0,0 +1,203 @@
+package org.robockets.runqueue.client.notifications;
+
+import java.io.*;
```
Comment (simon-andrews) [inline client/src/main/java/org/robockets/runqueue/client/notifications/MacNotification.java]: No star imports

Comment (jakekinsella) [inline client/src/main/java/org/robockets/runqueue/client/notifications/MacNotification.java]: Nooooo Will!

File: client/src/main/java/org/robockets/runqueue/client/notifications/MacNotification.java [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+/**
+ * Modified from https://github.com/n9Mtq4/MacOSX-Notification-Library-Java
```
Comment (simon-andrews) [inline client/src/main/java/org/robockets/runqueue/client/notifications/MacNotification.java:27]: Will's library is GPLv3. You must include the header that Will put in his code.

**Comment by simon-andrews** [comment]
I actually started ready about GPL on [StackOverflow](https://stackoverflow.com/questions/3902754/mit-vs-gpl-license). It turns out that if we distribute any GPL code then the entire project must be under GPL. Whoopee!

We could ask will nicely to relicense under the LGPL, or use a different library. If we want to relicense to GPL, we need to get the consent of every single person who has contributed code to this project. Personally I would be against it because I don't want to encourage the horrible disease on the open source community that is the GPL but if everyone else is okay with it then I can change my mind.

**Comment by jakekinsella** [comment]
Talking to Will now...

**Comment by Math-ias** [comment]
I have a good faith belief that use of the copyrighted materials described above as allegedly infringing is not authorized by the copyright owner, its agent, or the law.

I swear, under penalty of perjury, that the information in the notification is accurate and that I am the copyright owner or am authorized to act on behalf of the owner of an exclusive right that is allegedly infringed.

Screw this repository: https://github.com/Team4761/Run-Queue/tree/notifications

**Pull request closed without merging**